docs/experiments-msmarco-doc.md

```diff
@@ -216,3 +216,4 @@ That's it!
 + Results reproduced by [@printfCalvin](https://github.com/printfCalvin) on 2021-04-11 (commit [`d808d4a`](https://github.com/castorini/anserini/commit/d808d4a50ee69f44493e19a87dc36c7eb99402a9))
 + Results reproduced by [@saileshnankani](https://github.com/saileshnankani) on 2021-04-26 (commit [`5781c87`](https://github.com/castorini/anserini/commit/5781c871db12f0e36139982fbf1c805cfec189ee))
 + Results reproduced by [@andrewyguo](https://github.com/andrewyguo) on 2021-04-29 (commit [`71f3ca6`](https://github.com/castorini/anserini/commit/71f3ca671faf6ddd7b0dea0a1e7f4b590a0a02a5))
++ Results reproduced by [@mayankanand007](https://github.com/mayankanand007) on 2021-05-04
```

Comment:
Please change the the commit id you used to reproduce results.

Reply:
which "commit ID" is this exactly? I wasn't sure about that

Reply:
Fixed.